artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -1567,6 +1581,10 @@ public long getDurableScheduledSize() {
       return scheduledDeliveryHandler.getScheduledReferences();
    }
 
+   private String getSessionManagementString(ServerSession serverSession) {
```

Comment:
?

Reply:
I want to know the client ip that closes consumer but not committing session, so I added server session info to return.

Reply:
It forces string concatenation here though un-neededly.

Reply:
I copied code from Consumer::toManagementString. Do you mean using StringBuilder? It sounds not a hot path here.

Reply:
More the issue is that its used as a key later in code. Key should really be based off a unique id. Not some formatted string

Reply:
Session name is a unique id and is used as a key in ActiveMQServerImpl::sessions map.

Reply:
We should just be using that, not some new formatted key

Reply:
Formatting should be either in the control or owned by session so that its uniform anywhere its used

Reply:
See consumer.toManagementString()

Reply:
Checking some other code that displays management string representation exists this should follow suit so its uniform.